### PR TITLE
Add Checkpoint PSU status MIB

### PR DESCRIPTION
**Pull request by montecas**
Adding a new MIB to monitor the status of the PSUs in the Checkpoint Chassis

### CODE REVIEW

File: profiles/kentik_snmp/checkpoint/checkpoint-firewall.yml [hunk 1]
```diff
@@ -151,3 +151,15 @@ metrics:
...
+      - column:
+          OID: 1.3.6.1.4.1.2620.1.6.7.9.1.1.1
+          name: powerSupplyIndex
+        tag: sensor_index
+    symbols:
+      - OID: 1.3.6.1.4.1.2620.1.6.7.9.1.1.2
+        name: powerSupplySensorStatus
+    table:
+      OID: 1.3.6.1.4.1.2620.1.6.7.9
+      name: powerSupplySensorTable
```
Comment (thezackm) [inline profiles/kentik_snmp/checkpoint/checkpoint-firewall.yml:165]: @montecas - thanks for the submission; this section has some indentation and missing conversion errors that need to be resolved, and we'd like to keep the order of elements in line with the rest of the repo's patterns

```
table
symbols
metric_tags
```

suggested change:

```yaml
  - MIB: CHECKPOINT-MIB
    table:
      name: powerSupplySensorTable
      OID: 1.3.6.1.4.1.2620.1.6.7.9
    symbols:
      # this OID returns a 'DisplayString', we need to convert it to a metric
      - name: powerSupplySensorStatus
        OID: 1.3.6.1.4.1.2620.1.6.7.9.1.1.2
        conversion: to_one
```

1. indent added
2. re-order sections
3. add conversion to handle OID that returns a string value
4. remove index from `metric_tag` (we always collect the index, you don't need to call it out)

**Review by thezackm — CHANGES_REQUESTED** [review]

File: profiles/kentik_snmp/checkpoint/checkpoint-firewall.yml (hunk 1 above)
Comment (montecas) [reply profiles/kentik_snmp/checkpoint/checkpoint-firewall.yml:165]: This works for me, thank you for the feedback. Shall I update my PR with this or will you pull in the suggested changes?

Comment (thezackm) [reply profiles/kentik_snmp/checkpoint/checkpoint-firewall.yml:165]: please make the updates; thanks!

**Comment by montecas** [comment]
Updated

**Review by thezackm — APPROVED** [review]

**Pull request merged**